docs/config.md

```diff
@@ -33,9 +33,18 @@ The list of images can be separated by commas or by newlines.
   value: img1@sha256:digest1, img2@sha256:digest2
 ```
 
-When processing a `TaskRun`, Chains will parse through the list, then sign and attest each image.
+Chains also suports signing SBOMs. If using the `IMAGE_URL` and `IMAGE_DIGEST` combination, the
```

Comment:
Chains would be signing/attesting to something that it doesn't generate itself. While this is better than no signing, would it be worth it to call out that an individual may want to prefer to have the SBOM signed by its generator (if possible)?

The preferred approach would involve additional work in the deployment including the possibility of leveraging keyless signing and spiffe/spire.

Reply:
+1

I do think the same argument applies to singing images. Chains is signing something that it didn't generate. Could you file an issue?

Reply:
I cannot resolve this thread, but feel free to do so as I created https://github.com/tektoncd/chains/issues/791.

Reply:
Thank you!